boards/arm/nrf52_adafruit_feather/doc/nrf52_adafruit_feather.rst

```diff
@@ -0,0 +1,174 @@
+`Adafruit Feather nRF52 Bluefruit Learning Guide`_. The `Nordic Semiconductor Infocenter`_
+contains the processor's information and the datasheet.
+
+Hardware
+********
+
+- nRF52832 ARM Cortex-M4F processor at 64 MHz
+- 32.768 kHz crystal oscillator
+- 512 KiB flash memory and 64 KiB of SRAM
+- Battery connector and charger for 3.7 V lithium polymer batteries
+- Charging indicator LED
+- 2 User LEDs
+- Reset button
+- SWD connector
+- USB serial converter
+
+Supported Features
+==================
+
+The nRF52 Adafruit Feather board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| NVIC      | on-chip    | nested vectored      |
+|           |            | interrupt controller |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial port          |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+
+Connections and IOs
+===================
+
+The `Adafruit Feather nRF52 Bluefruit Learning Guide`_ has detailed
+information about the board including `pinouts`_ and the `schematic`_.
+
+LED
+---
+
+* LED0 (red) = P0.17
+* LED1 (blue) = P0.19
+
+Push buttons
+------------
+
+* DFU = SW0 = P0.20
+* RESET = SW1 = P0.21/reset
+
+Programming and Debugging
+*************************
+
+Applications for the ``nrf52_adafruit_feather`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+
+#. Build the Zephyr kernel and the :ref:`hello_world` sample application:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: nrf52_adafruit_feather
+      :goals: build
+      :compact:
+
+#. Connect the Adafruit nRF52 Feather to your host computer using USB
+
+#. Run your favorite terminal program to listen for output. 
```

Comment:
Looks like there's a stray space on the end of this line that needs to be deleted.